docs/content/gs-adding-users-proc.adoc

```diff
@@ -4,26 +4,43 @@ ifdef::env-github[]
 :imagesdir: ../images/
 endif::[]
 
-= Managing Users
 
-You manage users in the context of an Red Hat Single Sign-On realm.
-A realm manages a set of users, credentials, roles, and groups.
-A user belongs to and logs into a realm.
-Realms are isolated from one another and can only manage and authenticate the users that they control.
+= About Single Sign-On instances
 
-// removed realm as per https://issues.jboss.org/browse/INTLY-2168
-// https://gist.github.com/finp/ac8b7a743917b2b8268f28e73797e861
+:rhmi-sso-name: Managed Integration RH-SSO instance
+:customer-sso-name: End user RH-SSO instance
 
-== Accessing the Red Hat Single Sign-On administration console
+{PRODUCT_SHORT} provides two RH-SSO instances:
 
-To access the administration console navigate to the following URL, replacing `<cluster-host>` with the hostname of your cluster:
+* *{rhmi-sso-name}* - protects the Red Hat applications available from the Solution Explorer.
++
+For example, *{rhmi-sso-name}* defines the identities of the integration engineers and developers creating a solution for end users.
++
+Red Hat manages and provides limited access to this instance, for example, you can implement user federation with this instance, but you cannot create a realm.
++
+See xref:managing-users[Managing Users] and xref:gs-adding-custom-idp[Adding custom identity providers] sections for more information on using this instance.
 
-----
-https://sso-openshift-sso.apps.<cluster-host>/auth/admin/openshift/console
-----
+* *{customer-sso-name}* - protects the solutions developed using Managed Integration.
++
+Red Hat manages this instance, however the customer-admin user has admin level privileges to configure this instance.
++
+See the link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on[Product Documentation for Red Hat Single Sign-On] for more information on using Red Hat Single Sign-On.
+
+[NOTE]
+====
+Red Hat recommends that you define users of the solution you are developing in the *{customer-sso-name}*. 
+For example, if you use {PRODUCT_SHORT} to develop an solution named `my-app`, define all users of `my-app` in the *{customer-sso-name}* instance only.
+
+Typically, you only use the *{customer-sso-name}* for end users, however you have the required privileges to use it for developer related activities.
```

Comment:
This line seems vague and might prompt questions about what is meant by `developer related activities`.
What's the thinking behind having it?

Reply:
there was a thread around using http://microcks.github.io/ 
so, while it's called 'end user sso', it is also where developers might work.

would it help to add ", for example, if a developer wanted to use http://microcks.github.io/[Microcks], that developer could create associated users in this instance"?

Reply:
@david-martin  updated these paragraphs, and added upstream doc, could you give it another gander? thx